.changeset/famous-hairs-compete.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-css': minor
+'@microsoft/atlas-site': minor
+---
+
+Add new tab-container custom element
```

Comment:
This one should be segmented control, right?

Reply:
✅